### PR TITLE
Update Reactive Search/Native Websites

### PR DESCRIPTION
**Pull request by bietkul**

### CODE REVIEW

File: site/.eslintrc.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
+module.exports = {
+	extends: ['airbnb', 'plugin:jest/recommended'],
+	plugins: ['jest'],
+	env: {
+		browser: true,
+	},
+	parser: 'babel-eslint',
+	rules: {
```
Comment (divyanshu013) [inline site/.eslintrc.js]: Not all rules are required here

File: site/.eslintrc.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+		'no-underscore-dangle': 0,
+		camelcase: 0, // shopify and appbase data includes _
+
+		'react/jsx-indent': 0,
+		'react/jsx-indent-props': 0,
+		'react/jsx-filename-extension': 0,
+		'react/forbid-prop-types': 0,
+		'react/require-default-props': 0,
+	},
+	settings: {
```
Comment (divyanshu013) [inline site/.eslintrc.js]: `settings` are also not needed here I think.

File: site/404.html [hunk 3]
```diff
@@ -1,12 +1,13 @@
...
 <html>
+
 <head>
 	<meta charset="utf-8">
 	<meta http-equiv="X-UA-Compatible" content="IE=edge">
 	<meta name="viewport" content="width=device-width, initial-scale=1">
 	<title>Reactivesearch - UI components for Elasticsearch</title>
 	<link href="https://fonts.googleapis.com/css?family=Rubik:400,700" rel="stylesheet">
-	<link rel="stylesheet" href="styles.css">
+	<link rel="stylesheet" href="/styles.css">
```
Comment (divyanshu013) [inline site/404.html]: This could cause an issue on gh-pages since it will look at the root instead of /reactivesearch

File: site/404.html [hunk 4]
```diff
@@ -94,46 +121,51 @@
...
 		</svg>
 	</a>
 
 	<a href="https://www.producthunt.com/posts/reactivesearch-native" target="_blank" class="featured-post-link">
 		<span>Featured on Producthunt</span>🎉
 	</a>
 
-	<script src="dist/common.js"></script>
-	<script src="dist/main.js"></script>
+	<script src="/dist/common.js"></script>
```
Comment (divyanshu013) [inline site/404.html]: Here also not sure if this will work on gh-pages

File: site/index.html [hunk 5]
```diff
@@ -1,33 +1,61 @@
...
-	<link rel="apple-touch-icon" sizes="114x114" href="images/favicon/apple-icon-114x114.png">
-	<link rel="apple-touch-icon" sizes="120x120" href="images/favicon/apple-icon-120x120.png">
-	<link rel="apple-touch-icon" sizes="144x144" href="images/favicon/apple-icon-144x144.png">
-	<link rel="apple-touch-icon" sizes="152x152" href="images/favicon/apple-icon-152x152.png">
-	<link rel="apple-touch-icon" sizes="180x180" href="images/favicon/apple-icon-180x180.png">
-	<link rel="icon" type="image/png" sizes="192x192"  href="images/favicon/android-icon-192x192.png">
-	<link rel="icon" type="image/png" sizes="32x32" href="images/favicon/favicon-32x32.png">
-	<link rel="icon" type="image/png" sizes="96x96" href="images/favicon/favicon-96x96.png">
-	<link rel="icon" type="image/png" sizes="16x16" href="images/favicon/favicon-16x16.png">
+	<link rel="stylesheet" href="/styles.css">
```
Comment (divyanshu013) [inline site/index.html]: Here also the `/` might cause issue on gh-pages

Comment (bietkul) [reply site/index.html]: If we change it, then style is not working for nested routes

Comment (divyanshu013) [reply site/index.html]: Need to check

**Pull request merged**